### PR TITLE
fix(filelogreceiver): fix changing fingerprint_size

### PR DESCRIPTION
**Pull request by sumo-drosiek**
Signed-off-by: Dominik Rosiek <drosiek@sumologic.com>

### CODE REVIEW

File: otelcolbuilder/.otelcol-builder.yaml [hunk 1]
```diff
@@ -176,3 +176,14 @@ replaces:
...
+  # This fixes panic for changing fingerprint_size with already calculated fingerprint
+  # TODO: remove this when the following change is released upstream:
+  #
+  # - fix(pkg/stanza/input/file/reader): skip building fingerprint in case of configuration change
+  #   https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/10485
+  #
+  # For now, cherry-picked changes are being used:
+  # commit: https://github.com/SumoLogic/opentelemetry-collector-contrib/commit/6528176f3fd636ac770c5ebf1417718e84a7280e
+  # branch: https://github.com/SumoLogic/opentelemetry-collector-contrib/tree/v0.52.0-stanza
+  - github.com/open-telemetry/opentelemetry-collector-contrib/pkg/stanza => github.com/SumoLogic/opentelemetry-collector-contrib/pkg/stanza 6528176f3fd636ac770c5ebf1417718e84a7280e
```
Comment (astencel-sumo) [inline otelcolbuilder/.otelcol-builder.yaml:189]: Another fork to maintain 😞 Hopefully not for long?

**Review by astencel-sumo — APPROVED** [review]

File: otelcolbuilder/.otelcol-builder.yaml (hunk 1 above)
Comment (swiatekm-sumo) [reply otelcolbuilder/.otelcol-builder.yaml:189]: Only until the fix makes its way upstream.

Comment (sumo-drosiek) [reply otelcolbuilder/.otelcol-builder.yaml:189]: I will update `release.md`, just in case :sweat_smile:

Comment (sumo-drosiek) [reply otelcolbuilder/.otelcol-builder.yaml:189]: updated as `easy-to-drop` change, so not ideal, but easy to drop :D

**Pull request merged**